src/Infrastructure/Identity/ApplicationUser.cs

```diff
@@ -12,4 +13,6 @@ public class ApplicationUser : IdentityUser
     public DateTime RefreshTokenExpiryTime { get; set; }
 
     public string? ObjectId { get; set; }
+
+    public bool HasRootEmailAddress => Email == MultitenancyConstants.Root.EmailAddress;
```

Comment:
not need [NotMapped] because property can only get

Reply:
Ok, if you're sure then it can be removed...